.github/workflows/bot.yml

```diff
@@ -283,6 +284,11 @@ jobs:
     strategy:
       matrix:
         include:
+          - flinkProfile: 'flink1.18'
+            sparkProfile: 'spark3.4'
+            sparkRuntime: 'spark3.4.0'
+          - flinkProfile: 'flink1.18'
+            sparkProfile: 'spark3.3'
```

Comment:
Do we need to keep two items for flink1.17?

Reply:
I think it is not required; we could have one workflow for flink1.17 with the latest Spark 3.4 alone.